runtime/src/main/java/org/corfudb/protocols/wireprotocol/orchestrator/AbstractWorkflow.java

```diff
@@ -12,24 +12,33 @@
  * Created by Maithem on 10/25/17.
  */
 
-public interface Workflow {
+public abstract class AbstractWorkflow {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Convert the abstract class "AbstractWorkflow" into an interface. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1610)

Reply:
Now that sonarqube mentions it, why is this abstract? It has no fields.

Reply:
Changed to interface.